src/app/containers/ATIAnalytics/index.test.jsx

```diff
@@ -47,19 +51,17 @@ describe('ATI Analytics Container', () => {
 
       renderer.create(
         <ContextWrap platform="canonical" pageType="article">
-          <ATIAnalytics data={mockData} />
+          <ATIAnalytics data={articleDataNews} />
         </ContextWrap>,
       );
 
       expect(mockCanonical).toHaveBeenCalledTimes(1);
-      expect(mockCanonical).toHaveBeenCalledWith(
-        {
-          pageviewParams: mockAtiQueryParams,
-        },
-        mockData,
-      );
+      expect(mockCanonical.mock.calls[0][0]).toEqual({
```

Comment:
curious why you needed to change the test to this from `toHaveBeenCalledWith(`?

Reply:
resolved irl

Reply:
discussed offline - the test is asserting on how react interally renders components which is 🤮 so I've dropped my objections to this as it is easier to follow than the deleted code